### PR TITLE
Notes on local data and iOS permissions

### PR DESCRIPTION
**Pull request by const86**

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C6%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701062241524861-155%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/AppSDK/PlatformConsiderations.md [hunk 1]
```diff
@@ -40,7 +40,28 @@ DevicesManager audio interfaces are reduced on iOS. When headphones or a headset
 
 ### Permissions and security
```
Comment (JohnAustin-MSFT) [inline Skype/AppSDK/PlatformConsiderations.md]: We discuss permissions for Android apps in the Getting Started topic. Think about moving the permissions and security section to https://github.com/OfficeDev/skype-docs/blob/master/Skype/AppSDK/GettingStarted.md

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C10%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201701100048294912-155%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by const86** [comment]
@JohnAustin-MSFT, done.

**Pull request merged**